internal/integration/en_api.go

```diff
@@ -0,0 +1,160 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package integration
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+	"testing"
+
+	exportapi "github.com/google/exposure-notifications-server/internal/export"
+	"github.com/google/exposure-notifications-server/internal/pb/export"
+	"github.com/google/exposure-notifications-server/internal/util"
+	verifyapi "github.com/google/exposure-notifications-server/pkg/api/v1alpha1"
+)
+
+const appPackageName = "com.example.android.test"
+
+func collectExportResults(t *testing.T, exportDir string) *export.TemporaryExposureKeyExport {
+	exportFile := getExportFile(exportDir, t)
+
+	t.Logf("Reading keys data from: %v", exportFile)
+
+	blob, err := ioutil.ReadFile(exportFile)
+	if err != nil {
+		t.Fatalf("can't read export file: %v", err)
+	}
+
+	keyExport, err := exportapi.UnmarshalExportFile(blob)
+	if err != nil {
+		t.Fatalf("can't extract export data: %v", err)
+	}
+
+	return keyExport
+}
+
+type EnServerClient struct {
+	client *http.Client
+}
+
+// Posts requests to the specified url.
+// This methods attempts to serialize data argument as a json.
+func (server EnServerClient) postRequest(url string, data interface{}) (*http.Response, error) {
```

Comment:
Nit: Can `server` be `enServer` for consistency with below?

Reply:
Done.